homeassistant/components/androidtv/__init__.py

```diff
@@ -132,9 +151,19 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Android TV platform."""
 
     state_det_rules = entry.options.get(CONF_STATE_DETECTION_RULES)
-    aftv, error_message = await async_connect_androidtv(
-        hass, entry.data, state_detection_rules=state_det_rules
+    exceptions = (
+        ADB_PYTHON_EXCEPTIONS
+        if CONF_ADB_SERVER_IP not in entry.data
+        else ADB_TCP_EXCEPTIONS
```

Comment:
Please avoid the use of multi-line ternary. Use a normal if/else in such cases.

../Frenck

Reply:
The problem is that when I use normal if/else:
```
    if CONF_ADB_SERVER_IP not in entry.data:
        exceptions = ADB_PYTHON_EXCEPTIONS
    else:
        exceptions = ADB_TCP_EXCEPTIONS
```

I receive an error from mypy during commit:

Reply:
The issue is the adb_shell package, not exposing typing correctly. This makes HA assume everything is `Any` from what originates from adb_shell.